main_routers/system_router.py

```
@@ -3226,6 +3226,17 @@ async def _fetch_meme_with_fallback(kw: str):
                                 await mgr.feed_tts_chunk(cleaned)
                             continue
                         
+                        # --- 在线拦截: [PASS] ---
+                        if '[PASS]' in content.upper():
+                            # 部分模型会把 [PASS] 嵌在文本中输出
+                            content = re.sub(r'\[PASS\]', '', content, flags=re.IGNORECASE).strip()
+                            if content:
+                                full_text += content
+                                await mgr.feed_tts_chunk(content)
+                            print(f"[{lanlan_name}] Phase 2 流式检测到内嵌 [PASS]，abort")
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)